### ISSUE TITLE
Archive this Repo

### ISSUE THREAD

**Issue by RobLoach** [libretro/libretro-thumbnails#364]
Its been moved over to:
https://github.com/libretro-thumbnails/libretro-thumbnails

You can archive this old repo over at https://github.com/libretro/libretro-thumbnails/settings . May be good to add a note about it in the repo description too.

**Comment by twinaphex** [comment]
Hold on, are these new repos actually being used on the thumbnails server?

**Comment by twinaphex** [comment]
This is kinda confusing too. Why does this main repo still exist then?

**Comment by RobLoach** [comment]
Yes, https://github.com/libretro-thumbnails/libretro-thumbnails is active on:
http://thumbnailpacks.libretro.com

This repository exists because it has yet to be Archived...

![https://user-images.githubusercontent.com/125011/32273119-0fc5571e-bef9-11e7-9909-d137268a1d6d.png](https://user-images.githubusercontent.com/125011/32273119-0fc5571e-bef9-11e7-9909-d137268a1d6d.png)

If you add me as an Admin, that's something I could take on. Otherwise, feel free to do it yourself :wink:
